src/index.ts

```diff
@@ -1,273 +1,248 @@
+  });
+
+  const canvasPlaceholder = document.getElementById("canvas");
+  canvasPlaceholder.replaceWith(app.canvas);
+  await Assets.load(RouterSvg);
+  await Assets.load(ServerSvg);
+  await Assets.load(ComputerSvg);
+
+  // Context initialization
+  const ctx = new GlobalContext();
+
+  // Background container initialization
+  const viewport = new Viewport(app.renderer.events);
+  app.stage.addChild(viewport);
+
+  // Left bar logic
+  const leftBar = LeftBar.getFrom(document);
+
+  // Add router button
+  leftBar.addButton(RouterSvg, () => {
+    AddRouter(ctx); // Esta es una función anónima que ejecuta AddRouter cuando se hace clic
+  });
+
+  // Add server button
+  leftBar.addButton(ServerSvg, () => {
+    AddServer(ctx); // Función anónima que ejecuta AddServer cuando se hace clic
+  });
+
+  // // Add PC button
+  leftBar.addButton(ComputerSvg, () => {
+    AddPc(ctx); // Función anónima que ejecuta AddPc cuando se hace clic
+  });
+
+  // Get right bar
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const rightBar = RightBar.getFrom(document);
+
+  ctx.initialize(viewport);
+
+  // Ticker logic
+  // app.ticker.add(() => { });
+
+  // Resize logic
+  function resize() {
+    // Obtener los tamaños actuales de las barras desde el DOM
+    const leftBarWidth = lBar ? lBar.offsetWidth : 100; // Ancho actual de la barra izquierda
+    const rightBarWidth = rBar ? rBar.offsetWidth : 250; // Ancho actual de la barra derecha
+    const topBarHeight = tBar ? tBar.offsetHeight : 40; // Altura actual de la barra superior
+  
+    // Calcular el nuevo tamaño del canvas
+    const newWidth = window.innerWidth - leftBarWidth - rightBarWidth;
+    const newHeight = window.innerHeight - topBarHeight;
+  
+    // Redimensionar el renderer y el viewport de Pixi.js
+    app.renderer.resize(newWidth, newHeight);
+    viewport.resize(newWidth, newHeight);
+  }
+  
+  resize();
+
+  window.addEventListener("resize", resize);
+
+ // Gestión de los botones de carga y guardado
+ const loadButton = document.getElementById("load-button");
+ const saveButton = document.getElementById("save-button");
+
+ // Función para manejar el botón de guardar
+ saveButton.onclick = () => {
+   saveGraph(ctx); // Llama a la función saveGraph pasando el contexto actual
+ };
+
+ // Función para manejar el botón de cargar
+ loadButton.onclick = () => {
+   const input = document.createElement("input");
+   input.type = "file";
+   input.accept = ".json";
+
+   input.onchange = (event) => {
+     const file = (event.target as HTMLInputElement).files[0];
+     const reader = new FileReader();
+     reader.readAsText(file);
+
+     reader.onload = (readerEvent) => {
+       const jsonData = readerEvent.target.result as string;
+       loadGraph(jsonData, ctx); // Llama a la función loadGraph pasando el JSON y el contexto
+     };
+   };
+
+   input.click(); // Simula el click para abrir el cuadro de diálogo de selección de archivo
+ };

```

Comment:
The `saveButton` and `loadButton` not being simmetric smells. This should either go inside `loadGraph`, or we should take out the save-to-file logic inside `saveGraph`.